documentation/release-process.md

```diff
@@ -97,11 +97,9 @@ The nightly image is `mcr.microsoft.com/dotnet/nightly/monitor`. The tag list is
 ## Release to nuget.org and Add GitHub Release
 
 1. Run the [Generate release notes](https://github.com/dotnet/dotnet-monitor/actions/workflows/generate-release-notes.yml) workflow, setting `Use workflow from` to the release branch. Review and merge in the PR created by this workflow.
-1. Start [release pipeline](https://dev.azure.com/dnceng/internal/_release?_a=releases&view=mine&definitionId=105). Allow the stages to trigger automatically (do not check the boxes in the associated dropdown). During creation of the release you must select the dotnet-monitor build to release from the list of available builds. This must be a build with the tag `MonitorRelease` and the associated `MonitorRelease` artifact (set `dotnet-monitor_build` to the pipeline run of `dotnet monitor` that is being released; set `dotnet-monitor_source` to the latest commit from `main`).
+1. **The day of the release** run the [dotnet-dotnet-monitor-publish](https://dev.azure.com/dnceng/internal/_build?definitionId=1444) pipeline. During creation of the release you must select the dotnet-monitor build to release from the list of available builds. This must be a build with the tag `MonitorRelease` and the associated `MonitorRelease` artifact. Set the branch to `main`. Set `DraftRelease` to false when ready to publish.
```

Comment:
> Set the branch to `main`.

The `main` branch is the default of the pipeline, correct? I don't think this needs to be called out. And if you do leave it in, it should be moved to the front of the instructions after mentioning which pipeline it is because the pipeline resources and parameters will be reset after changing the branch.